[2021-03-12 11:23:51,149] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: conversion_flow.taskUploadResult 2021-03-12T11:23:18+00:00 [queued]>
[2021-03-12 11:23:51,159] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: conversion_flow.taskUploadResult 2021-03-12T11:23:18+00:00 [queued]>
[2021-03-12 11:23:51,159] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-12 11:23:51,159] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-12 11:23:51,159] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-12 11:23:51,164] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): taskUploadResult> on 2021-03-12T11:23:18+00:00
[2021-03-12 11:23:51,165] {standard_task_runner.py:52} INFO - Started process 3177 to run task
[2021-03-12 11:23:51,168] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'conversion_flow', 'taskUploadResult', '2021-03-12T11:23:18+00:00', '--job-id', '16', '--pool', 'default_pool', '--raw', '--subdir', '/srv/app/src/ckanext-mysql2mongodb/ckanext/mysql2mongodb/data_conv/dag/conversion_flow.py', '--cfg-path', '/tmp/tmpdllng5tv', '--error-file', '/tmp/tmpqf61enpr']
[2021-03-12 11:23:51,168] {standard_task_runner.py:77} INFO - Job 16: Subtask taskUploadResult
[2021-03-12 11:23:51,194] {logging_mixin.py:104} INFO - Running <TaskInstance: conversion_flow.taskUploadResult 2021-03-12T11:23:18+00:00 [running]> on host 6299f34f884f
[2021-03-12 11:23:51,216] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=conversion_flow
AIRFLOW_CTX_TASK_ID=taskUploadResult
AIRFLOW_CTX_EXECUTION_DATE=2021-03-12T11:23:18+00:00
AIRFLOW_CTX_DAG_RUN_ID=conversion_flow1615548198.567747
[2021-03-12 11:23:51,469] {python.py:118} INFO - Done. Returned value was: None
[2021-03-12 11:23:51,476] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=conversion_flow, task_id=taskUploadResult, execution_date=20210312T112318, start_date=20210312T112351, end_date=20210312T112351
[2021-03-12 11:23:51,490] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-12 11:23:51,502] {local_task_job.py:146} INFO - Task exited with return code 0
